components/textbox/textbox.tsx

```diff
@@ -39,6 +38,7 @@ type Props = {
     onBlur?: (e: FocusEvent) => void;
     supportsCommands: boolean;
     handlePostError?: (message: JSX.Element | null) => void;
+    suggestionList?: React.Component<React.ComponentProps<typeof SuggestionList>>;
```

Comment:
Didn't realize this could be simplified last round 😛 (also fixes a type)
```suggestion
    suggestionList?: typeof SuggestionList;
```

Reply:
Oh that's much easier. I know I've used typeof here a few times, but I didn't actually understand how it worked in a typing context.

Reply:
Okay, I wasn't able to get this to work. I started getting some bizarre errors from AppsFormField when I tried to set this type, so I gave up and essentially set them back to any. Assuming this passes the CI checks finally, I think we might just need to live with this for now.

I don't know if `typeof SuggestionList` works when you're wanting any component with the same props as `SuggestionList`, but perhaps that's because there's so much plain JS code in between that definition and the code using it. We might need to wait until all of this is migrated to be able to resolve those issues.

Reply:
Big +1 on letting it be with a now-passing CI.